CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project are documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+## [v1.3.0] 2020-02-14
+
+### Added
+
+- **CUMULUS-1896**
+  - Added a 'streaming' mode to the CMA.   This will allow client libraries to avoid requiring three subprocess/python invocations when running the CMA commands in sequence, thus lowering
+    the overhead of utilizing the CMA.
+
+### BREAKING CHANGES
```

Comment:
I would put the breaking changes at the top of the version notes

Reply:
f1272be